src/components/attendee/InputSelect/InputSelect.module.scss

```
@@ -0,0 +1,32 @@
+@use "scss/attendee/variables.scss"as *;
+
+.InputSelect {
+  &__wrapper {
```

In modules there is no need to make complicated class names to avoid name clashes, so another, a KISS, way to do this:

```suggestion
  .wrapper {
```

For this and any other instances. We can make the classes flat or as close to flat as we can, to simplify use `styles.wrapper` vs `styles.InputSelect__wrapper`

---

done [04e9125](https://github.com/sparkletown/sparkle/pull/2840/commits/04e91251fb1d5bfc42be24e1adab66087443d930)